config/php-cs-fixer.php

```diff
@@ -2,154 +2,96 @@
 
 declare(strict_types=1);
 
-if (PHP_SAPI !== 'cli') {
-    die('This script supports command line usage only. Please check your command.');
-}
-
 return (new \PhpCsFixer\Config())
     ->setRiskyAllowed(true)
     ->setRules(
         [
-            // rule sets
-            '@DoctrineAnnotation' => true,
-            '@PHP56Migration:risky' => true,
-            '@PHP70Migration' => true,
-            '@PHP70Migration:risky' => true,
-            '@PHP71Migration' => true,
-            '@PHP71Migration:risky' => true,
+            '@PER-CS2.0' => true,
+            '@PER-CS2.0:risky' => true,
+
+            '@PHPUnit50Migration:risky' => true,
+            '@PHPUnit52Migration:risky' => true,
+            '@PHPUnit54Migration:risky' => true,
+            '@PHPUnit55Migration:risky' => true,
+            '@PHPUnit56Migration:risky' => true,
             '@PHPUnit57Migration:risky' => true,
             '@PHPUnit60Migration:risky' => true,
             '@PHPUnit75Migration:risky' => true,
             '@PHPUnit84Migration:risky' => true,
-            '@PER' => true,
-
-            // alias
-            'no_alias_functions' => true,
 
-            // array notation
-            'array_syntax' => ['syntax' => 'short'],
-            'no_trailing_comma_in_singleline' => true,
-            'no_whitespace_before_comma_in_array' => true,
-            'whitespace_after_comma_in_array' => true,
-
-            // basic
-            'encoding' => true,
-            'psr_autoloading' => true,
+            // overwrite the PER2 defaults to restore compatibility with PHP 7.x
+            'trailing_comma_in_multiline' => ['elements' => ['arrays']],
 
             // casing
             'magic_constant_casing' => true,
             'native_function_casing' => true,
 
             // cast notation
-            'cast_spaces' => ['space' => 'none'],
-            'lowercase_cast' => true,
             'modernize_types_casting' => true,
             'no_short_bool_cast' => true,
-            'short_scalar_cast' => true,
 
             // class notation
-            'class_attributes_separation' => true,
-            'no_blank_lines_after_class_opening' => true,
             'no_php4_constructor' => true,
 
-            // class usage
-            // (no rules used from this section)
-
             // comment
             'no_empty_comment' => true,
-            'single_line_comment_style' => true,
-
-            // constant notation
-            // (no rules used from this section)
 
             // control structure
-            'elseif' => true,
             'no_superfluous_elseif' => true,
```

Comment:
If we remove `no_useless_else`, we should also remove `no_superfluous_elseif`.

Reply:
Done.